js/apps/admin-ui/src/App.tsx

```diff
@@ -2,15 +2,13 @@ import type KeycloakAdminClient from "@keycloak/keycloak-admin-client";
 import { Page } from "@patternfly/react-core";
 import type Keycloak from "keycloak-js";
 import { PropsWithChildren, Suspense } from "react";
-import { ErrorBoundary } from "react-error-boundary";
```

Comment:
If this is the only place this is package was being used I believe we can remove it from the package file.

Reply:
we are using useErrorHandler in AdminClient.tsx

Reply:
useErrorHandler from react-error-boundary is being use in useFetch inside AdminClient.tsx. Should I log the error or throw the error if I remove the useErrorHandler?

[https://github.com/keycloak/keycloak/blob/main/js/apps/admin-ui/src/context/auth/AdminClient.tsx](url)